addons/web_editor/static/src/js/editor/snippets.editor.js

```diff
@@ -594,6 +596,17 @@ var SnippetEditor = Widget.extend({
         }
         return show;
     },
+    /**
+     * @private
+     * @param {Object} $el
+     */
+    _getActivatedTarget: function ($el) {
+        let $elToActivate = $el.prev();
+        if (!$elToActivate.length) {
+            $elToActivate = $el.next().length ? $el.next() : $el.parent();
```

Comment:
Don't call `$el.next()` twice.
Also: try to not use jQuery at all for this task, you really don't need it.